sdk/servicebus/perf-tests/service-bus-track-1/test/receiveBatch.spec.ts

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ReceiveMode, Receiver, Sender } from "@azure/service-bus";
+import { PerfOptionDictionary } from "@azure/test-utils-perf";
+import { ServiceBusTest } from "./sbBase.spec";
+
+interface ReceiverOptions {
+  "max-message-count": number;
+  "number-of-messages": number;
+  "message-body-size-in-bytes": number;
+}
+
+/**
+ * An underestimate of the true maximum size of a batch send message in bytes.
+ * This number is used to split up the send requests into small enough chunks
+ * to be accepted.
+ */
+const MAX_SEND_BATCH_SIZE_BYTES = 200000;
+
+export class BatchReceiveTest extends ServiceBusTest<ReceiverOptions> {
+  receiver: Receiver;
+  public options: PerfOptionDictionary<ReceiverOptions> = {
+    "number-of-messages": {
+      required: true,
+      description: "Total number of messages to send",
+      shortName: "max",
+      longName: "maximum",
+      defaultValue: 10000,
+    },
+    "message-body-size-in-bytes": {
+      required: true,
+      description: "Size of each message body in bytes",
+      shortName: "size",
+      longName: "size-in-bytes",
+      defaultValue: 2000,
+    },
+    "max-message-count": {
+      required: true,
+      description: "Max number of messages to receive",
+      shortName: "max",
+      longName: "maximum",
+      defaultValue: 50,
+    },
+  };
+
+  constructor() {
+    super();
+    this.receiver = ServiceBusTest.sbClient
+      .createQueueClient(BatchReceiveTest.queueName)
+      .createReceiver(ReceiveMode.receiveAndDelete);
+  }
+
+  /**
+   * Sends the messages to be received later.
+   */
+  public async globalSetup(): Promise<void> {
+    await super.globalSetup();
+    const sender = ServiceBusTest.sbClient
+      .createQueueClient(BatchReceiveTest.queueName)
+      .createSender();
+
+    const {
+      "number-of-messages": { value: numberOfMessages },
+      "message-body-size-in-bytes": { value: messageBodySize },
+    } = this.parsedOptions;
+
+    await sendMessages(sender, numberOfMessages, messageBodySize);
+  }
+
+  public async runBatch(): Promise<number> {
+    const messages = await this.receiver.receiveMessages(
+      this.parsedOptions["max-message-count"].value,
+      0.5
```

Comment:
make it a parameter?

Reply:
Just following the lead from the track 2 test where it isn't a parameter.